app/controllers/submissions_controller.rb

```diff
@@ -549,9 +549,18 @@ def view
       @is_pdf = false
     end
 
-    respond_to do |format|
-      format.html { render(:view) }
-      format.js
+    begin
+      respond_to do |format|
+        format.html { render(:view) }
+        format.js
+      end
+    rescue StandardError
```

Comment:
Is this error handling still required? I suppose it helps with UX but is not strictly required?

Reply:
Yeah it's not really required anymore, I'll see if removing it is fine.

Reply:
Yeah I guess with the changes to code_viewer we don't need to do the error handling

Reply:
Actually I take it back, some files still cause an error when trying to initially view the assessment so this error handling is required.

Reply:
Hmm, what kind of files are not handled?

Reply:
oh nvm nvm actually I'm getting an error due to #1616 I think haha so we can remove